guide/general/priority_list.md

```diff
@@ -99,9 +99,16 @@ Covenant | <input type="radio" id="ft-radio" name="cov" value="1" /><label for="
     - {{ site.data.spell.es }}
     - {{ site.data.spell.lb }}
 
+**Tip**
+When using { site.data.lengendary.windspeaker }}, {{ site.data.talent.sk }} and {{ site.data.talent.mote }} on single target. You can go {{ site.data.spell.lava_burst }}, {{ site.data.spell.lb }} (empowered by storm keeper), {{ site.data.spell.lava_burst }}, {{ site.data.spell.es }}, {{ site.data.spell.lava_burst }} (from WLR), {{ site.data.spell.lb }} (empowered by storm keeper)
```

Comment:
Stormkeeper should be available as `{{ site.data.talent.stormkeeper }}`, too. Generally everything that references a spell, talent, or active effect like legendary should be a link.

Same goes for the "(from WLR)". Please make WLR a link, too